tools/greptimedbcluster/dump-data/dump-data.sh

```
@@ -0,0 +1,196 @@
+    for i in "${!values[@]}"; do
+      len=${#values[i]}
+      if [ "$len" -gt "${max_widths[i]:-0}" ]; then
+        max_widths[$i]=$len
+      fi
+    done
+  done <<< "$rows"
+
+  # Generate the separator line
+  divider="+"
+  for w in "${max_widths[@]}"; do
+    divider="$divider$(printf '%*s' $((w+2)) | tr ' ' '-')+"
```

Shellcheck (0.10.0)

[warning] This format string has 2 variables, but is passed 1 argument.

(SC2183)